src/javascript/binary/static/profittable.js

```diff
@@ -0,0 +1,14 @@
+
+pjax_config_page("profit_tablews", function(){
+    return {
+        onLoad: function() {
+            Content.populate();
```

Comment:
well, there's some terrible dependency problem
1. since we want to have all text together, they sits in the Content.localize object
2. Content.localize object is initialized when populate is called, it cannot be initialized when script is executed, because by then the global text might not be available
3.  so to ensure the text in Content.localize is accessible, populate must be called after global text is available and before my page use them, which happen to be here

Any suggestion is welcome =)